xbmc/Application.cpp

```diff
@@ -296,6 +296,7 @@ CApplication::CApplication(void)
   , m_fallbackLanguageLoaded(false)
   , m_WaitingExternalCalls(0)
   , m_ProcessedExternalCalls(0)
+  , m_ProcessedExternalDecay(0)
```

Comment:
I meant doing init by assignment in the header. that is much better than having to keep the right order in the constructor.